client/app/views/admin/users.html

```diff
@@ -162,7 +162,8 @@
                   </div>
                   <div class="form-group">
                     <label data-translate>Password</label>
-                    <input name="password" class="form-control" data-ng-model="user.password" maxlength="{{node.maximum_namesize}}" type="password" />
+                    <input name="password" class="form-control" data-ng-model="user.password" maxlength="{{node.maximum_namesize}}" type="password" zxcvbn="passwordStrength" />
```

Comment:
Why keep a max password length? If you are looking for strong passwords why let the node admin force them to be weak?